CHANGES.rst

```diff
@@ -4,6 +4,12 @@ Changelog for zest.releaser
 4.0 (unreleased)
 ----------------
 
+- Added support for twine_.  If the ``twine`` command is available, it
+  is used for uploading to PyPI.  It is added to the ``recommended``
```

Comment:
"It is added by the recommended extra" => "It is installed automatically if you use `zest.releaser[recommended]` extra".